shared/bundle_analysis/parser.py

```diff
@@ -123,6 +126,20 @@ def parse(self, path: str) -> int:
             e.bundle_analysis_plugin_name = self.info.get("plugin_name", "unknown")
             raise e
 
+    def _asset_type(self, name: str) -> AssetType:
+        extension = get_extension(name)
+
+        if extension in ["js"]:
```

Comment:
just curious, where did we compile this list of extensions?

Reply:
this matches how Sentry buckets its asset types.
Or defined here https://www.notion.so/sentry/Type-Filtering-and-Grouping-82d5db9fd7f84737a4792df2ae30c22f

Reply:
perf! Thanks, exactly what I was looking for